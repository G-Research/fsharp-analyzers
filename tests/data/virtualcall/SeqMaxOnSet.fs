module M

let f () =
    let mySet = set [1..10]
    Seq.max mySet // should warn
