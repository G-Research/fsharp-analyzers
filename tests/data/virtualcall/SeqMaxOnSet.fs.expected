GRA-VIRTUALCALL-001 | Warning | (5,4 - 5,17) | Consider replacing the call of Seq.max with a function from the Set module to avoid the costs of virtual calls. | [(5,4 - 5,11) Set.maxElement]
